modules/basics/src/main/java/com/opengamma/strata/basics/ReferenceData.java

```diff
@@ -47,7 +48,22 @@ public static ReferenceData of(Map<? extends ReferenceDataId<?>, ?> values) {
    * @return standard reference data
    */
   public static ReferenceData standard() {
-    return StandardReferenceData.INSTANCE;
+    return StandardReferenceData.STANDARD;
+  }
+
+  /**
+   * Obtains the minimal set of reference data.
+   * <p>
+   * The {@linkplain #standard() standard} reference data contains common holiday calendars
+   * and indices, but may not be suitable for production use. The minimal reference data contains
+   * just those identifiers that are needed by Strata, and that are non-controversial.
+   * These are {@link HolidayCalendars#NO_HOLIDAYS}, {@link HolidayCalendars#SAT_SUN},
+   * {@link HolidayCalendars#FRI_SAT} and {@link HolidayCalendars#THU_FRI}.
+   *
+   * @return minimal reference data
+   */
+  public static ReferenceData minimal() {
```

Comment:
This PR only solves half the problem when creating reference data which doesn't use the default holidays. In order to create reference data the code will look something like this:

    ReferenceData userRefData = ReferenceData.of(...);
    ReferenceData refData = userRefData.combinedWith(ReferenceData.minimal());

That's not a particularly obvious API. Why not simply include `NO_HOLIDAYS` in the reference data created with `ReferenceData.of()`? Strata doesn't work without `NO_HOLIDAYS` so I can't see a reason for wanting to create a set of reference data that doesn't include it.